packages/signers/src/capsule/types.ts

```diff
@@ -0,0 +1,53 @@
+import type { Environment } from "@usecapsule/web-sdk";
+import type { WalletClientConfig } from "viem";
+
+// copied since Capsule doesn't export this type
```

Comment:
@nityas is there a way for Capsule to export these types from the capsule sdk so we don't have to re-write them here?

Reply:
you could do something like `Params<typeof Capsule.prototype.constructor>` or w/e class you need the constructor params of